types/stringify-object/stringify-object-tests.ts

```diff
@@ -1,4 +1,4 @@
-import stringifyObject, { Options } from 'stringify-object';
+import { default as stringifyObject, type Options } from 'stringify-object';
```

Comment:
```suggestion
import stringifyObject from 'stringify-object';
// tslint:disable-next-line:no-duplicate-imports testing imports
import type { Options } from 'stringify-object';
```

Reply:
The whole point of this change is to test it understands `import { default as SymbolName } from "module-name"`, because typescript interprets `import SymbolName from "module-name"` pretty liberally. Although it may not be he case with `type: "module"` in place.